lesson2/index.html

```diff
@@ -9,14 +9,20 @@
 	<script type="text/javascript" src="presentationsPage/presentationsPage.js"></script>
 	<script type="text/javascript" src="chatPage/chatPage.js"></script>
 
+	<script type="text/javascript" src="router.js"></script>
 	<script type="text/javascript" src="app.js"></script>
 </head>
 <body>
 <div class="main">
 	<div class="header">
-		<div class="menu-btn">&#9776;</div>
+		<div class="menu-btn" onclick="this.parentNode.classList.toggle('hidden');">&#9776;</div>
```

Comment:
I would avoid usage of inliners here.
in short:
 - it's better to keep js outside of html
- if your application will change html dynamically those inline onclick event handlers won't be removed from the memory.
If you like to have more details I reccomend this old but good article: https://robertnyman.com/2008/11/20/why-inline-css-and-javascript-code-is-such-a-bad-thing/